molstar/example-gallery
# Delete representation and subtree visibility example

## Overview
This PR creates 2 new examples for deleting representations and setting the visibility of a StateObject subtree.

## Changes made
- Created representation example "delete_representations"
  - This example shows you how to delete representations you have created
- Created representation example "visibility_using_tags"
  - This example shows you how to set the visibility of a StateObject subtree based on tags

## How to test
Ensure the representation is deleted
  - [delete_representations](https://codesandbox.io/p/sandbox/github/molstar/example-gallery/tree/delete-representation/representation/delete_representations)
Ensure the water representation is hidden
  - [visibility_using_tags](https://codesandbox.io/p/sandbox/github/molstar/example-gallery/tree/delete-representation/representation/visibility_using_tags)

## Specific review requests
For each of the examples:
- Are there enough comments to sufficiently describe the process?
- Are the comments accurately describing the process?
- Are there any anti-patterns in the examples?
- Can the examples be more efficient?
